### DISCUSSION TITLE
How to display details using foriegn keys?

### DISCUSSION THREAD

**Discussion by SwastikGorai** [aminalaee/sqladmin#555] [Q&A] [answered]
![Screenshot 2023-07-19 230833](https://github.com/aminalaee/sqladmin/assets/43789832/a0f3c006-246e-4684-96f6-1cd9db22195a)

Here, I want to display the test names corresponding to the `test_id`. The test name is from a Model known as `Mrp`. This models stores the associated `test_id`. So, instead of showing the `test_id`, I'd like to show the names of it, from the `Test` model. Is there anyway to do it?

**Comment by sorix6** [comment]
I think you can use custom views https://aminalaee.dev/sqladmin/writing_custom_views/

**Comment by aminalaee** [comment, marked as answer]
Right now there is no easy way to do this, but after this change we can start it: https://github.com/aminalaee/sqladmin/pull/544
